### DISCUSSION TITLE
Why doesn't `player-commands` include the command `tpacancel`?

### DISCUSSION THREAD

**Discussion by hojongs** [EssentialsX/Essentials#4800] [Q&A] [answered]
The players have the tpa command in their default permissions in default config.yml.
But, they can't use the tpacancel which cancels their own request that they sent.

Why isn't included In the default config.yml?

**Comment by pop4959** [comment, marked as answer]
The `player-commands` configuration is an old and seldom used feature of Essentials and the list often isn't up-to-date with newer commands and features. We generally recommend that you use a permission plugin on your server, as this list also only pertains to Essentials commands, so you are not able to manage other plugin commands with it.

**Reply by hojongs** [reply]
Thank you for your answer, that was the reason EssentialsX strongly recommends using aother permission plugin. I see.
I could solve the issue with the LuckPerms plugin! Thank you.
